driver/collector/postgres_collector.py

```diff
@@ -138,15 +140,27 @@ def __init__(
         """
         self._conn = conn
         self._version_str = version
-        if float(".".join(version.split(".")[:2])) >= 9.4:
-            # pylint: disable=invalid-name
+        version_float = float(".".join(version.split(".")[:2]))
+        # pylint: disable=invalid-name
```

Comment:
what was the error here and why are we okay with it?

Reply:
We are using Upper letters here, e.g., `PG_STAT_VIEWS`,  `PG_STAT_STATEMENTS_SQL`. I think it should be fine because we are using the same format for all such sql queries